functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
Yep! Thanks, so much and welcome to the team, @DevOpsFu! Thank you for your awesome code 💯